opensearch-operator/pkg/builders/cluster.go

```diff
@@ -221,8 +222,9 @@ func NewSTSForNodePool(
 	}
 	mainCommand := helpers.BuildMainCommand("./bin/opensearch-plugin", cr.Spec.General.PluginsList, true, startUpCommand)
 
-	initContainers := []corev1.Container{
-		{
+	var initContainers []corev1.Container
+	if os.Getenv(helpers.SkipInitContainerEnvVariable) != "true" {
```

Comment:
There is `strconv.ParseBool` to parse a string to a bool. That allows the users to use different variants of `true`.
Also please move the env lookup part into `helpers/constants.go` (like already done with `ParallelRecoveryMode` and `ClusterDnsBase`).